pkg/utils/dataset.go

```diff
@@ -71,6 +72,25 @@ func GetAccessModesOfDataset(client client.Client, name, namespace string) (acce
 
 }
 
+func GetPVCStorageCapacityOfDataset(client client.Client, name, namespace string) (storageCapacity resource.Quantity, err error) {
+	dataset, err := GetDataset(client, name, namespace)
+	if err != nil {
+		return storageCapacity, err
+	}
+	annotations := dataset.GetObjectMeta().GetAnnotations()
+	size := annotations["pvc.fluid.io/resources.requests.storage"]
+	if size == "" {
+		storageCapacity = resource.MustParse("100Pi")
+		return
+	}
+	storageCapacity, err = resource.ParseQuantity(size)
+	if err != nil {
+		storageCapacity = resource.MustParse("100Pi")
+		return
```

Comment:
If err is not nil, then is the expected behavior to return error or using default value?